crates/bevy_ui/src/layout_components.rs

```diff
@@ -0,0 +1,314 @@
+#![warn(missing_docs)]
+
+//! Components used to control the layout of [`UiNode`] entities.
+use crate::{Size, UiRect, Val};
+use bevy_derive::{Deref, DerefMut};
+use bevy_ecs::prelude::{Bundle, Component};
+use bevy_reflect::prelude::*;
+use serde::{Deserialize, Serialize};
+
+/// Controls which layout algorithm is used to position this UI node
+#[derive(
+    Component, Copy, Clone, PartialEq, Eq, Debug, Default, Serialize, Deserialize, Reflect,
+)]
+#[reflect_value(PartialEq, Serialize, Deserialize)]
+pub enum LayoutStrategy {
+    /// Use the absolute position and size specified
+    None,
+    /// Use the [Flexbox](https://cssreference.io/flexbox/) layout algorithm
+    ///
+    /// As implemented by [`taffy`]: some bugs or limitations may exist; please file an issue!\
+    #[default]
+    Flexbox,
+}
+
+/// The strategy used to position this node
+#[derive(
+    Component, Copy, Clone, PartialEq, Eq, Debug, Default, Serialize, Deserialize, Reflect,
+)]
+#[reflect_value(PartialEq, Serialize, Deserialize)]
+pub enum PositionType {
+    /// Relative to all other nodes with the [`PositionType::Relative`] value
+    #[default]
+    Relative,
+    /// Independent of all other nodes
```

Comment:
```suggestion
    /// Positioned as if it was the only child of its parent
```
Not sure if this is clearer?